packages/fast-breakpoint-tracker-react/README.md

```diff
@@ -0,0 +1,40 @@
+# React Breakpoint Tracker
+`fast-breakpoint-tracker-react` is a React component for managing viewport breakpoints. Given a set of breakpoints, this component will monitor the width of the browser window and track the current active breakpoint.
+
+## Installation
+`npm i --save @microsoft/fast-breakpoint-tracker-react`
+
+## Breakpoints
+`Breakpoints` are expected as an array. Six default breakpoint values are provided for convenience:
+
+```jsx
+const defaultBreakpoints = [0, 540, 768, 1084, 1400, 1779];
```

Comment:
Is the first array element `0` a valid breakpoint?  Feels like the first one might be 320 or 540.  Not zero.  Don't we assume zero to some breakpoint?

Reply:
This matches our previous implementations and breakpoints from a mobile first strategy.